consensus/parlia/parlia.go

```diff
@@ -891,6 +891,10 @@ func (p *Parlia) Seal(chain consensus.ChainHeaderReader, block *types.Block, res
 				log.Info("Received block process finished, abort block seal")
 				return
 			case <-time.After(time.Duration(processBackOffTime) * time.Second):
+				if chain.CurrentHeader().Number.Uint64() == header.Number.Uint64() {
```

Comment:
how about >=?
it could happen, in case:
- local mined has unstable network
- it mined a off-turn block, with backOffTime =4, say: T0 + 4
- its network resumed at T0 + 3, it could import 2 in-turn blocks simultaneously
- then, currentHeader would > header

Reply:
I think >= is better

Reply:
It's not how it works.
There's a `delay` logic before this `shouldWaitCheck`, so it only would happen when T0-Block started to insert before T0+4, and this offturn validator would wait to T0+5 for the `verifiedHeader` block's body/state be verified and check whether it is `updated as canonical`. and it's not possible for T0+6 block imported(future block)

Reply:
It's not how it works.
There's a delay logic before this shouldWaitCheck, so it only would happen when T0-Block started to insert before T0+4, and this offturn validator would wait to T0+5 for the verifiedHeader block's body/state be verified and check whether it is updated as canonical. and it's not possible for T0+6 block imported(future block)

Reply:
It's not how it works.
There's a delay logic before this shouldWaitCheck, so it only would happen when T0-Block started to insert before T0+4, and this offturn validator would wait to T0+5 for the verifiedHeader block's body/state be verified and check whether it is updated as canonical. and it's not possible for T0+6 block imported(future block)

Reply:
might insist use `==`, since `>` would happen on another fork